snap/snapcraft.yaml

```diff
@@ -98,5 +98,17 @@ apps:
   z88dk-font2pv1000:
     command: bin/z88dk-font2pv1000
     plugs: [home, removable-media]
+  z88dk-asmpp.pl:
+    command: bin/z88dk-asmpp.pl
+    plugs: [home, removable-media]
+  z88dk-asmstyle.pl:
+    command: bin/z88dk-asmstyle.pl
+    plugs: [home, removable-media]
+  z88dk-zsdcc:
```

Comment:
There's no need to expose zsdcc and zsdcpp outside of the snap.

Reply:
Got it. Backed out those 2 files.